### ISSUE TITLE
Hello @pcarret, thanks for contacting us.

### ISSUE THREAD

**Issue by pcarret** [dotnet/aspnetcore#39205]
Hello @pcarret, thanks for contacting us. 

When running inside Visual Studio, potentially with the debugger attached, and/or using a debug build, the GC behaves differently. Consequently, these results are likely not representative of how Blazor behaves in a production environment. 

If you want to capture an accurate trace of your program we suggest you follow some of the guidance provided [here](https://docs.microsoft.com/aspnet/core/performance/memory) to accurately measure the memory usage in your app.

_Originally posted by @TanayParikh in https://github.com/dotnet/aspnetcore/issues/39156#issuecomment-999782563_

**Comment by pcarret** [comment]
I have checked the guidance provided.
What can we conclude if Task Manager show that the exe memory increase like VS already shown.
I am not sure it's Visual Studio is hiding something and it seems something bad about memory is happening for no obvious reason. Do I need to consider this as an individual and localized issue having no interest to the community ?
I my end app the memory rasie from 88 Mo to 190 in one minute while user does simply zooming / unzooming operation where the math computation is done and call to js interop as well

I could throw away the interest and months I spent already in Blazor if only 20 users can use one server at same time
